supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala

```diff
@@ -367,7 +367,15 @@ final case class Input(name: Option[String],
                        path: String,
                        `type`: Option[String],
                        content: Option[String]
-)
+) {
+  override def toString: String = {
+    import akka.http.scaladsl.model.Uri
+
+    // Remove query that may contain SAS token
+    val cleanUrl = url map { Uri(_).copy(rawQueryString = None).toString() }
```

Comment:
What do you think about only stripping out query params when we can tell the URL contains a SAS token?

Also, can we replace with `?REDACTED` or similar so we can tell something was removed? I can imagine users or our future selves not realizing this is happening and thinking Cromwell is incorrectly stripping the query params before passing the URL to TES.

Reply:
I was able to enhance and adopt an existing library function for this!